package.json

```diff
@@ -15,7 +15,9 @@
     "registry": "https://artifacts.aue1d.saasure.com/artifactory/api/npm/npm-okta"
   },
   "scripts": {
-    "lint:report": "grunt jshint",
+    "lint": "eslint ./lib ./test",
+    "lint:fix": "eslint --fix ./lib ./test",
+    "lint:report": "eslint -f checkstyle -o build2/loginjs-checkstyle-result.xml ./lib ./test",
```

Comment:
Yeah we recently standardized on the build/reports once we started doing modules in the npm repo.